packages/admin-ui/src/components/Filter/components/FilterCheckbox/ChecklistFilterContent.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef } from 'react'
+
+import type { UseFilterCheckboxReturn } from './useFilterCheckbox'
+
+import { Checkbox } from '../../../Checkbox'
+import { useOption } from '@react-aria/listbox'
+import { tag } from '@vtex/admin-ui-react'
+import { focusVisible } from '@vtex/admin-ui-core'
+import { Set } from '../../../Set'
+import type { FilterItem } from '../Filter'
+
+export function ChecklistFilterContent(props: FilterCheckboxProps) {
+  const {
+    state: { listBoxProps, listState, ref },
+  } = props
+
+  return (
+    <Set
+      as="ul"
+      spacing={5}
+      orientation="vertical"
+      ref={ref}
+      csx={{ margin: '$l', paddingY: '$m' }}
+      {...listBoxProps}
+    >
+      {[...listState.collection].map((item) => {
+        return <Option key={item.key} item={item} state={listState} />
+      })}
+    </Set>
+  )
+}
+
+function Option({ item, state }: any) {
```

Comment:
That's interesting. The only thing that changes from the MultipleSelection and Single, is this component. 🤔